packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -417,7 +417,14 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
       // Normally we can bail out on props equality but if context has changed
       // we don't do the bailout and we have to reuse existing props instead.
     } else if (memoizedProps === nextProps) {
-      return bailoutOnAlreadyFinishedWork(current, workInProgress);
+      if (
```

Comment:
renderExpirationTime === Offline?

Reply:
There's a test that fails when I do that, for some reason, haven't figured out why yet. Seems like it should do the same thing, since it will end up bailing out a few lines below, anyway.

Reply:
Ah figured it out